packages/cf.js/API_REFERENCE.md

```diff
@@ -156,7 +156,7 @@ Params:
 
 Result:
 - `appInstanceId: string`
-    - Unique ID for this app instance
+    - ID of an app instance whose proposed installation was accepted
```

Comment:
@IIIIllllIIIIllllIIIIllllIIIIllllIIIIll This isn't true. The installation isn't yet accepted when this ID is returned

Reply:
Ah yeap, this should be changed